.github/ISSUE_TEMPLATE/bugreport.yml

```diff
@@ -80,7 +80,7 @@ body:
     id: packages
     attributes: 
       label: Packages
-      description: List Discord.Net and Discord.Net related pacakges in use, and versions.
-      placeholder: Discord.Net V3.3.2, etc.
+      description: List related packages in use, and versions.
```

Comment:
'Please list all 3rd party packages in use if applicable, including their versions.' Sounds better to me, and resolves listing of packages that do not relate to Discord.NET.

Reply:
Agreed. That does sound better. Changed.